query-engine/js-connectors/js/pg-js-connector/src/conversion.ts

```diff
@@ -0,0 +1,51 @@
+import { ColumnTypeEnum, type ColumnType } from '@jkomyno/prisma-js-connector-utils'
+import { types } from 'pg'
+
+const NeonColumnType = types.builtins
```

Comment:
copy/paste or should this _really_ say `PgColumnType` or similar?

Reply:
copy/paste